stargate-starter/src/main/java/io/stargate/starter/Starter.java

```diff
@@ -288,6 +288,11 @@ void setStargateProperties() {
     }
   }
 
+  public Starter withAuthEnabled(boolean enabled) {
+    enableAuth = enabled;
```

Comment:
I'm a bit confused by this. It's only used to set PasswordAuthenticator and CassandraAuthorizer on the Cassandra daemon that we start in the persistence module.

Is that a separate mechanism from auth-table-based-service? When would you use both?

Reply:
auth-table-based-service relies on CassandraAuthorizer as it converts auth tokens to C* roles.